torch/csrc/jit/python/python_arg_flatten.cpp

```diff
@@ -62,6 +63,9 @@ void flatten_rec(PyObject* obj, ParsedArgs& args) {
     args.vars.push_back(var);
     args.desc.metadata.emplace_back(var);
     args.desc.structure.push_back(D::Variable);
+  } else if (strcmp(THPUtils_typename(obj), "NoneType") == 0) {
```

Comment:
curious what happens if None appears inside a tuple e.g. model((x, None), y). from the code structure here looks like it is already supported?

Reply:
Added test for this